.storybook/stories/tooltip.stories.mdx

```diff
@@ -26,7 +48,22 @@ import { Tooltip } from "../misc";
     </Story>
 </Canvas>
 
-<ArgsTable of={TooltipWrapper} />
+## Dynamically Left Positioned
```

Comment:
Either "Dynamic Left Positioned" or change line 34 for consistent titles. 
I also think a brief description here would be helpful since the naming conventions are a bit confusing at first pass. Maybe "A tooltip with the `tooltipPosition` prop set to `left` that does not have enough available space to the left."